packages/app/integration-tests/tests/sandboxes.test.js

```diff
@@ -13,7 +13,7 @@ const SANDBOXES = [
   'o29j95wx9',
   'k3q1zjjml5',
   'github/reduxjs/redux/tree/master/examples/real-world',
-  'github/CompuIves/codesandbox-presentation',
+  'github/codesandbox/codesandbox-presentation',
```

Comment:
```suggestion
  'github/CompuIves/codesandbox-presentation',
```

This one needs to stay the same, that one is still on my personal profile 😄

Reply:
Oops 😓